scripts/bert/index.rst

```diff
@@ -279,3 +279,168 @@ Command line interface
        -0.42823148, -0.48873612], dtype=float32), array([-0.6576557 , -0.09822252,  0.1121515 , ..., -0.21743725,
        -0.1820574 , -0.16115054], dtype=float32)]
 
+
+Example Usage of Exporting Hybridizable BERT
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The BERTModel class is a subclass of Block, rather than HybridBlock.
+To support exporting BERT model to json format for deployment, we introduce the StaticBERT class.
+Specifically, by exporting hybridizable BERT, we mean the BERT with fixed input embedding size and sequence length can be exported through
+a static shape based implementation of hybridblock based BERT. By using this, we can export a block based BERT model.
+
+Please follow the steps below for exporting the model.
+
+
+Step 1: create a hybridizable task guided model using BERT:
+
+.. code-block:: console
+
+    $ class StaticBertForQA(HybridBlock)
+
+An example can be found in 'staticbert/static_bert_for_qa_model.py'.
+
+
+Step 2: hybridize the model in the script:
+
+.. code-block:: console
+
+    $ net = StaticBertForQA(bert=bert)
+    $ net.hybridize(static_alloc=True, static_shape=True)
+
+An example can be found in 'staticbert/static_export_squad.py'.
+
+
+Step 3: export trained model:
+
+.. code-block:: console
+
+    $ net.export(os.path.join(args.output_dir, 'static_net'), epoch=args.epochs)
+
+To export the model, in 'staticbert/static_export_squad.py', set export=True.
+
+To run the example, if you would like to export the Block parameters
+and test the HybridBlock on your datasets with the specified input size and sequence length,
+
+.. code-block:: console
+
+    $ cd staticbert
+    $ python static_export_squad.py --model_parameters output_dir/net.params --export --evaluate --seq_length 384 --input_size 768 --gpu 0
+
+This will load the the StaticBERTQA HybridBlock with parameter (requirement: output_dir/net.params should exist)
+trained by a normal BERTQA Block, and export the HybridBlock to json format.
+
+Besides, Where seq_length stands for the sequence length of the input, input_size represents the embedding size of the input.
+
+
+Example Usage of Finetuning Hybridizable BERT
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example mainly introduces the steps needed to use the hybridizable BERT models to finetune on a specific NLP task.
+We use SQuAD dataset for Question Answering as an example.
+
+
+Step 1: create a hybridizable task guided model using BERT:
+
+.. code-block:: console
+
+    $ class StaticBertForQA(HybridBlock)
```

Comment:
same here. also, some contents here duplicate with the above section. consider referring to the previous section instead.